pkg/session/session.go

```diff
@@ -69,6 +67,10 @@ func (s *session) Disconnect() {
 	s.adminClient = nil
 	s.isAuthenticated = false
 	s.version = constants.SaramaKafkaVersion
+	s.isSecureKafka = false
```

Comment:
I think if you just do `s = session{}` it will be better. This way you don't have to worry about clearing all the properties from the session object when you want to log out